resources/packs/python/python.lr

```diff
@@ -0,0 +1,22 @@
+
+import "../core/core.lr"
+
+// Python packages resource
+python {
+  init(path? string)
+  path string
+  packages() []python.package
+  children() []python.package
```

Comment:
it feels weird to me to type `python.children` and know that it is about immediate packages. Semantically it could be child installations of python, possibly variants, runtimes, as well as packages. 

I do like `python.packages` for showing all installed packages

Reply:
A few ideas:
`python.packages.explicit` for explicitly installed ones (this is how pacman works in arch linux)
`python.packages.toplevel` for toplevel packages maybe
`python.packages.userinstalled` which is how dnf does it

Reply:
I ended up changing it to `toplevel`.